src/main/java/com/qdesrame/openapi/diff/model/ChangedApiResponse.java

```diff
@@ -43,6 +44,11 @@ public ChangedApiResponse(
 
   @Override
   public DiffResult isCoreChanged() {
+    ApiResponse oldDefault = oldApiResponses.getDefault();
+    ApiResponse newDefault = newApiResponses.getDefault();
+    if (!Objects.equals(oldDefault, newDefault)) {
```

Comment:
this doesn't feel quite right. the `Changed*` objects are just holder objects...  and you don't want exact equality (as mentioned in the bug below)

Reply:
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#response-object So should the comparison be: Equality of all subfields, except description?

Reply:
i don't know that its strict equality of all subfields (e.g., order of headers doesn't matter). I'm also not sure how `links` should be handled yet... these don't appear in the actual request/response body, but are annotations used for openapi processing itself.

I think somehow this needs to be used to compare the responses: https://github.com/confluentinc/openapi-diff/blob/master/src/main/java/com/qdesrame/openapi/diff/compare/ResponseDiff.java#L43